WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -48,6 +50,9 @@
             </td>
           <% end %>
 
+          <td>
+            <%= registration.events.length %>
```

Comment:
Yeah, I think it's still better. Note that even if we do eager loading so that we don't perform another query for the events, the `events` method perform the mapping anyway.